integration/env/common.go

```diff
@@ -86,31 +105,30 @@ func init() {
 		panic("version bundle version  must not be empty")
 	}
 	os.Setenv(EnvVarVersionBundleVersion, VersionBundleVersion())
+
+	// ClusterID returns a cluster ID unique to a run integration test. It might
+	// look like ci-w3e95.
+	//
+	//     ci is a static identifier stating a CI run of the aws-operator.
+	//     w is a version reference for wip which can also be c for the current version.
+	//     3 is the first character of the Git SHA.
+	//     e95 is a randomly generated alphanumeric string.
```

Comment:
This is necessary because the old IDs were too long. It is just luck IMO that this worked in the previous way for so long. If we ever put it into a AWS resource, it was close to 32 characters when in reality IDs are much shorter.

The solution here is a bit dirty with a random string, but works much more reliably so far.

Reply:
I don't get this. How `ci-wip-3cc75-5e958` is even close to 32 characters?

Reply:
Because we did fun stuff like `public-elb-<CLUSTER-ID>`.
This was the change that broke the camels back: https://github.com/giantswarm/aws-operator/pull/1940/files#r325820549

Note that this is totally fine in an actual cluster (and quite far away from 32 characters) but broke E2E.